django-server/server/api/views.py

```diff
@@ -112,6 +112,131 @@ def get_queryset(self):
         return FixedMenuOptionReview.objects.all().filter(option_id=self.kwargs['option_pk']).order_by('id')
 
 
+class TimeStampedFilterSet(filters.FilterSet):
+    timestamp_start = filters.DateTimeFilter(required=True)
```

Comment:
Can we add `from` and `to` filters? It seems this comes at no extra cost.

Reply:
added. the minimum of time_end, time.now(), count is taken into account when stats are generated